### PR TITLE
Copy-paste gist from Takhion's

### PR DESCRIPTION
**Pull request by tiwiz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,6 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+
+signing.properties
+droidconit.jks
```
Comment (rock3r) [inline .gitignore]: That missing empty line...

Comment (rock3r) [inline .gitignore]: Also :x: why is local.properties under VCS?

File: android/build.gradle [hunk 2]
```diff
@@ -217,8 +212,8 @@ android {
             buildConfigField "String", "FEEDBACK_SURVEY_ID", "\""+droidconit15_feedback_survey_id+"\""
             buildConfigField "String", "FEEDBACK_URL", "\""+droidconit15_feedback_url+"\""
             buildConfigField "String", "UNIQUE_CONTENT_IDENTIFIER", "\""+uniqueContentIdentifier+"\""
-            buildConfigField "String", "WIFI_SSID", "\""+ droidconit15_wifi_ssid +"\""
-            buildConfigField "String", "WIFI_PASSPHRASE", "\"" + droidconit15_wifi_password + "\""
+            buildConfigField "String", "WIFI_SSID", "\"droiconit\""
```
Comment (rock3r) [inline android/build.gradle:215]: :x: here too -  this should NOT be under VCS

File: .gitignore (hunk 1 above)
Comment (tiwiz) [inline .gitignore]: No idea, honestly

Comment (Takhion) [inline .gitignore]: Was about to say, but it's actually `.gitignore`... WTF?

Comment (rock3r) [inline .gitignore]: Please delete it and then we can go ahead with the next :x: below

File: android/build.gradle (hunk 2 above)
Comment (Takhion) [inline android/build.gradle:215]: TBF it will be "public", but yeah

File: .gitignore [hunk 3]
```diff
@@ -42,3 +42,5 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+signing.properties
+droidconit.jks
```
Comment (rock3r) [inline .gitignore]: I was about to say...  That's weird. @tiwiz please don't add files that aren't going to be published to the git ignore by name.

Comment (rock3r) [inline .gitignore]: (this also means the signing.properties will be loads of fun -  just learn to live with that uncommitted file)

File: android/build.gradle (hunk 2 above)
Comment (rock3r) [inline android/build.gradle:215]: It's a matter of consistency

File: .gitignore (hunk 3 above)
Comment (tiwiz) [inline .gitignore]: ALT+ENTER error... fixing

Comment (Takhion) [inline .gitignore]: I think `signing.properties` _should_ be in the `.gitignore` as it's named in the `build.gradle` and must never be committed

Comment (rock3r) [inline .gitignore]: Fair point

Comment (Takhion) [inline .gitignore]: @tiwiz for the keystore, put it in a folder (like `/keystore`), then inside that folder put a `.gitignore` that ignores everything (just a `*`), so also itself... :sparkles: magic :sparkles:

Comment (rock3r) [inline .gitignore]: Damn I'm on the phone and can't attach a magic.gif

Comment (Takhion) [inline .gitignore]: @rock3r let me fix that for you

MAGIC

**Comment by Takhion** [comment]
purrrfect

**Pull request merged**